File: level80/homework/codwars10.py
#https://www.codewars.com/kata/51edd51599a189fe7f000015/train/python
def solution(array_a, array_b):
    k=[]
    a=len(array_a)
    b=len(array_b)
    if a==b:
        for i in range(a):
            k.append((abs(array_a[i]-array_b[i])**2))
        l1=sum(k)
        r=l1/a
        return r

File: level 080/homework/codwars10.js
//https://www.codewars.com/kata/51edd51599a189fe7f000015/train/javascript
var solution = function(firstArray, secondArray) {

  var arr = [];

  for (var i = 0; i < firstArray.length; i++) {
    arr.push(Math.pow(secondArray[i] - firstArray[i], 2));
  }
  return arr.reduce((a,b) => a+b)/arr.length;

}